service/templates/mfa.html

```diff
@@ -15,6 +15,12 @@ <h1>
          <span>Enter MFA Token</span>
       </h1>
 
+      {% if tenant_id == "designsafe" %}
+      <p>
+         Multi-Factor Authentication (MFA) is now required. <small><a href="https://docs.tacc.utexas.edu/basics/mfa/" target="_blank">Set up MFA</a> via the <span>TACC&nbsp;User&nbsp;Portal</span>.</small>
```

Comment:
Message is shorter than [original request](https://github.com/tapis-project/authenticator/issues/80), but fear not:
- what is not in this message is in the docs is mentioned in docs
    <sup>e.g. [select pairing method](https://docs.tacc.utexas.edu/basics/mfa/#setupmfa-step2) mentions DesignSafe users can not use SMS</sup>
- this message only appears for DesignSafe (so no need to be explicit)